.github/workflows/release.yml

```diff
@@ -21,8 +15,6 @@ jobs:
             exit 1
           fi
       - uses: actions/checkout@v4
-        with:
-          ref: ${{ inputs.branch_specifier }}
```

Comment:
Choosing the branch from which to run the workflow is a built-in part of github. No need to duplicate that.